tests/e2e/tidbcluster/serial.go

```diff
@@ -746,6 +753,39 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			pdClient, cancel, err := proxiedpdclient.NewProxiedPDClient(c, fw, ns, clusterName, false, nil)
 			framework.ExpectNoError(err, "create pdapi error")
 			defer cancel()
+
+			// wait for ReadyToScaleThresholdSeconds seconds
+			klog.Info("wait 300 seconds, auto scale should not happen with noise reduction")
+			time.Sleep(300 * time.Second)
+
+			tc, err := cli.PingcapV1alpha1().TidbClusters(tc.Namespace).Get(tc.Name, metav1.GetOptions{})
+			if err != nil {
+				framework.ExpectNoError(err, "get tidb cluster error")
+			}
+			tac, err = cli.PingcapV1alpha1().TidbClusterAutoScalers(ns).Get(tac.Name, metav1.GetOptions{})
+			if err != nil {
+				framework.ExpectNoError(err, "get tidb cluster auto scaler error")
+			}
+			if tac.Annotations == nil || len(tac.Annotations) < 1 {
+				klog.Infof("tac haven't marked any annotation")
+				framework.ExpectNoError(err, "tac no annotation error")
+			}
+			t, ok := tac.Annotations[label.AnnTiKVReadyToScaleTimestamp]
+			if !ok {
+				klog.Info("tac has no tikv.tidb.pingcap.com/ready-to-scale-timestamp annotation")
+				framework.ExpectNoError(errors.NotFound(""), "tac has no tikv.tidb.pingcap.com/ready-to-scale-timestamp annotation error")
```

Comment:
`go-openapi/errors` is not suitable here (it generates HTTP not found error). you can simply use `framework.Failf(...)`